.github/codeowners.md

```diff
@@ -1,12 +1,12 @@
 # Hi! Welcome to the CODEOWNERS file.
 # These owners are the default reviewers for any change made to these subdirectories.  
 
-catbuffer/parser/	@Jaguar0625 @gimre-xymcity
-catbuffer/schemas/	@Jaguar0625 @gimre-xymcity
-client/catapult/	@Jaguar0625 @gimre-xymcity 
-client/rest/	@Jaguar0625 @gimre-xymcity
-linters/	@Jaguar0625 @gimre-xymcity @wayonb
-sdk/python/	@Jaguar0625 @gimre-xymcity 
-sdk/javascript/	@Jaguar0625 @gimre-xymcity 
-docs/	@0x6861746366574
-jenkins/	@wayonb
\ No newline at end of file
+catbuffer/parser/		@Jaguar0625 @gimre-xymcity
+catbuffer/schemas/		@Jaguar0625 @gimre-xymcity
+client/catapult/		@Jaguar0625 @gimre-xymcity 
+client/rest/			@Jaguar0625 @gimre-xymcity
+linters/				@Jaguar0625 @gimre-xymcity @wayonb
+sdk/python/			@Jaguar0625 @gimre-xymcity 
+sdk/javascript/		@Jaguar0625 @gimre-xymcity 
+docs/					@0x6861746366574
+jenkins/ 				@Wayonb
```

Comment:
I fucking give up on aligning it - GitHub, Sublime and cli all give different outputs. 
Also, for some reason @Wayonb's username is not being picked up by GitHub. 🤔

Reply:
'tabs for indents, spaces for alignment' ;)